app/services/cfe/create_explicit_remarks_service.rb

```diff
@@ -0,0 +1,22 @@
+module CFE
+  class CreateExplicitRemarksService < BaseService
+    delegate :legal_aid_application, to: :submission
+    delegate :policy_disregards, to: :legal_aid_application
+
+    def cfe_url_path
+      "/assessments/#{@submission.assessment_id}/explicit_remarks"
+    end
+
+    def request_body
+      {
+        explicit_remarks: [
+          policy_disregards.as_json
+        ]
+      }.to_json
+    end
+
+    def process_response
+      @submission.explicit_remarks_created!
```

Comment:
would it be worth having a guard clause.  
`@submission.explicit_remarks_created! unless @submission.explicit_remarks_created?`

only because they can go back and resubmit to CFE, if it fails.

Reply:
We don't on any of the other submission services... maybe with adding a ticket to discuss on each of the steps?